contentcuration/contentcuration/models.py

```diff
@@ -425,27 +427,17 @@ def save(self, *args, **kwargs):
         super(RelatedContentRelationship, self).save(*args, **kwargs)
 
 class Exercise(models.Model):
```

Comment:
This model is currently unused, but I'll keep it in here for now in case we need to use it again